### DISCUSSION TITLE
Disable  2FA

### DISCUSSION THREAD

**Discussion by ricky1146** [Kovah/LinkAce#805] [Support]
Hello, i was not able to find a similar post.  Is there a way to disable 2FA by ssh, or via the file? I have the recovery codes, but they do not seem to work.

**Comment by Kovah** [comment]
Hi, there is a console command for this: https://www.linkace.org/docs/v1/cli/#view-2fa-recovery-codes

**Reply by ricky1146** [reply]
Thanks, will try. I will love to see the 2.0 version, and multi users :)

**Reply by ricky1146** [reply]
@Kovah It seems the 2fa doesn't disable from the settings. I get 

{"checkResult":true}

When, i refresh, or logout it still seems to be turned on.

**Reply by Kovah** [reply]
If you have access to the database, you might also go the very rough way and delete the contents of the `two_factor_secret` and `two_factor_recovery_codes` column for your user.

**Reply by ricky1146** [reply]
hmm, i think the table is  deleted, by the system when i tried to disable it. Or perhaps  i've hit a bug? 
![Screen Shot 2024-07-30 at 2 55 06 PM](https://github.com/user-attachments/assets/d211b118-1b73-4048-be70-bf70e9176a44)

**Reply by Kovah** [reply]
Sorry, forgot to mention that the columns are in the `users` table.

**Reply by ricky1146** [reply]
I know this is a year later, but Thank you!  I finally got around to fixing this part. Now the question to upgrade from 1.14.2 to 2.x. I saw there is a composer issue that needs php 8.2 for 2.4.0.

I will decide about this upgrade

**Comment by ricky1146** [comment]
closing this as it is solved
